app/views/case_workers/admin/case_workers/_form.html.haml

```diff
@@ -1,37 +1,30 @@
 = form_for [:case_workers, :admin, @case_worker] do |f|
-
   = f.fields_for :user do |user_fields|
-    .form-fields
-      %fieldset
-        %legend.visually-hidden
-          = 'Case worker details'
-        .form-group
-          .form-group.input
-            = user_fields.label :first_name, t('.first_name'), class: "form-label"
-            = user_fields.text_field :first_name, class: "form-control"
-          .form-group.input
-            = user_fields.label :last_name, t('.last_name'), class: "form-label"
-            = user_fields.text_field :last_name, class: "form-control"
-          .form-group.input
-            = user_fields.label :email, t('.email'), class: "form-label"
-            = user_fields.email_field :email, class: "form-control"
-          .form-group.input
-            - if @case_worker.new_record?
-              = user_fields.label :email_confirmation, t('.email_confirmation'), class: "form-label"
-              = user_fields.email_field :email_confirmation, class: "form-control"
-
-      %fieldset
-        %legend
-          = 'Location and Roles'
+    %fieldset
+      %legend
+        = 'Case worker details'
+      .form-group
+        = user_fields.label :first_name, t('.first_name'), class: "form-label"
+        = user_fields.text_field :first_name, class: "form-control"
+      .form-group
+        = user_fields.label :last_name, t('.last_name'), class: "form-label"
+        = user_fields.text_field :last_name, class: "form-control"
+      .form-group
+        = user_fields.label :email, t('.email'), class: "form-label"
+        = user_fields.email_field :email, class: "form-control"
+      - if @case_worker.new_record?
         .form-group
-          = f.label :location_id, class: "form-label"
-          .form-group.inline
-            = f.collection_radio_buttons(:location_id, Location.all, :id, :name) do |b|
-              - b.label(class: "block-label") { b.radio_button + b.object.name }
+          = user_fields.label :email_confirmation, t('.email_confirmation'), class: "form-label"
+          = user_fields.email_field :email_confirmation, class: "form-control"
+      .form-group
+        = f.label :location_id, class: "form-label"
```

Comment:
Shouldn't a fieldset/legend be wrapped around radio buttons? 
http://govuk-elements.herokuapp.com/form-elements/#form-radio-buttons 
http://webaim.org/techniques/forms/#labels
http://webaim.org/techniques/forms/controls#radio